src/subword.rs

```diff
@@ -250,6 +322,19 @@ mod tests {
         };
     }
 
+    lazy_static! {
+        static ref NGRAMS_INDICES_TESTS_36: HashMap<&'static str, Vec<(&'static str, u64)>> = hashmap! {
+            "<Daniël>" =>
+                vec![("Dan",214157), ("iël",233912), ("Danië",311961), ("iël>",488897), ("niël>",620206), ("anië",741276), ("Dani",841219),
```

Comment:
Run `cargo fmt`.

It seems that CI doesn't fail with incorrect formatting, though it should. I'll fix that.

Reply:
Sorry, I'm not so sure which part of it is incorrect? I ran `cargo fmt` and `cargo fmt --all -- --check` on it and it turns out nothing's changed.

Reply:
Tuple elements have to be separated by a space (e.g. `("Dan", 214157)`). Also the lines are too long. `rustfmt` does not seem to fix these cases because they are in a structure-matching macro call.

Reply:
You could hand-format it, or replace it by something that `rustfmt` can format, e.g.:

```rust
    lazy_static! {
        static ref NGRAMS_INDICES_TESTS_36: HashMap<&'static str, Vec<(&'static str, u64)>> = [
            (
                "<Daniël>",
                vec![
                    ("Dan", 214157),
                    ("iël", 233912),
                    ("Danië", 311961),
                    ("iël>", 488897),
                    ("niël>", 620206),
                    ("anië", 741276),
                    ("Dani", 841219),
                    ("Daniël", 1167494),
                    ("ani", 1192256),
                    ("niël", 1489905),
                    ("ël>", 1532271),
                    ("nië", 1644730),
                    ("<Dan", 1666166),
                    ("aniël", 1679745),
                    ("<Danië", 1680294),
                    ("aniël>", 1693100),
                    ("<Da", 2026735),
                    ("<Dani", 2065822)
                ]
            ),
            (
                "<hallo>",
                vec![
                    ("lo>", 75867),
                    ("<hal", 104120),
                    ("hallo>", 136555),
                    ("hal", 456131),
                    ("allo>", 599360),
                    ("llo", 722393),
                    ("all", 938007),
                    ("<ha", 985859),
                    ("hallo", 1006102),
                    ("allo", 1163391),
                    ("llo>", 1218704),
                    ("<hallo", 1321513),
                    ("<hall", 1505861),
                    ("hall", 1892376)
                ]
            )
        ]
        .iter()
        .cloned()
        .collect();
    }
```
```

Reply:
Wow, thank you!!